### PR TITLE
Navbar Component

### LINKED ISSUE #24: feature/navbar
## User Story: (4 points)
As a customer I want to be able to navigate through the website's different pages, sign up, sign in and check my profile details.

## Brief Description
Create a responsive navigation bar that links to different pages

## Feature Requirements
-  add navbar
-  add platforms' name
-  add menu, cart, sign in and register buttons
-  add profile and profile pop-up options 

## Sub-tasks (sub-issues)
- [ ] #5 
- [x] #6 
- [ ] #10

### PR DESCRIPTION
**Pull request by emma-amar**
## Brief Description
Created the NavBar! Its purpose is to help users navigate the website. It also features the platform's logo and a profile feature for users who have signed up, have signed up to sell, or have not yet signed up and are just browsing. 

## Linked Issues
- closes #24 

## Changes
- changed the colour and position of the navbar in App.css
- added the profile icon
- added the profile menu and logic for each user (signed in, signed in and a seller, sign up)
- added the website branding
- linked the navbar in App.js

## Additional context
Modifications were made to the look of the website along with the navbar's own style changes.

### CODE REVIEW

**Comment by aspyridakos** [comment]
@emma-amar I started the routing for you. Search still needs to be implemented but product link works now at least.

File: online-shopping-website/src/App.css [hunk 1]
```diff
@@ -24,7 +29,7 @@
 }
 
 .RoutingLink {
-  text-decoration: none;
```
Comment (aspyridakos) [inline online-shopping-website/src/App.css:27]: Removed `color: white` on merge.

Cannot modify this css label as it is used globally. You'd have to manually override it for the component you need this for.

**Comment by aspyridakos** [comment]
I managed to find an improvised way of creating a shrinking menu however the image link still needs fixing for the small bobble logo (I think I messed up the image path route).

File: online-shopping-website/src/App.css (hunk 1 above)
Comment (emma-amar) [reply online-shopping-website/src/App.css:27]: this isn’t present in the latest stuff 😎

**Comment by emma-amar** [comment]
> I managed to find an improvised way of creating a shrinking menu however the image link still needs fixing for the small bobble logo (I think I messed up the image path route).

i was able to fix this !! thanks again for the menu !!!

**Review by Adamo-O — DISMISSED** [review]

**Review by aspyridakos — APPROVED** [review]
Love this! 11/10 🚀

**Pull request merged**